openapi/openapi.yml

```diff
@@ -997,6 +944,59 @@ paths:
           $ref: '#/components/responses/Error'
       tags:
         - Postgres Database
+  '/project/{project_id}/location/{location}/postgres/{postgres_database_reference}/failover':
```

Comment:
We removed failover endpoints from the codebase with [this PR](https://github.com/ubicloud/ubicloud/pull/2923), FYI